src/components/menu/menu.ts

```diff
@@ -113,6 +104,8 @@ export class Menu extends Component<Props, SpreadsheetChildEnv> {
   private menuRef = useRef("menu");
   private position: DOMCoordinates = useAbsoluteBoundingRect(this.menuRef);
 
+  static timeOut: any = null;
+
```

Comment:
I got some strange behaviour  https://drive.google.com/file/d/1CEoeNLlOpgrgLs1KN4YpjqvdzY6IWbAO/view?usp=drive_link

the timeout should probably be cancelled if the mouse leaves the element